### DISCUSSION TITLE
cool photography app but could use some improvements

### DISCUSSION THREAD

**Discussion by yzak4815** [KillerInk/FreeDcam#142] [Ideas]
I am a professional of digital dslr photography that is experiencing the possibilities of modern cameras implemented in mobile phones. This app beats all the ones I have tried by a lot, both proprietary and open source, this app meets the objective but I think it lacks some things that would come in handy in professional use.
1: the camera interface is very good but the icons are very small, it would be nice if they were a little bigger, not much, maybe double for the little ones.
2: the exposure time is limited to 1/3 of a second, it is very little in mobile cameras that have low iso sensitivity, mounting the mobile on a tripod, it would be fabulous to be able to take pictures with longer exposure, this is very useful for photography nocturnal and nocturnal time-lapses.

Apart from that, I find the app very good, the best I have used and now I will use it as a tool in my professional work, both photo and video, I hope my economy improves soon and so I can donate something to this valuable project. I write this hoping that the developer sees it, and hopefully it is possible to include changes to improve future versions.
I know that the project is open source, but I know very little about app development, I cannot modify the code due to my ignorance on the subject.

**Comment by KillerInk** [comment]
Hi.
About 1:
You mean the icons from the bottom bar? IMHO they are big enough. But a option can get added that increase the size

About2:
Ae is most time limited to 1/3 to prevent motion. There is nothing i can do.
If you need longer expotimes you have to use the manuals. But there its also up to phone wich limits can get used.
Some support only 1 sec, others 30sec.
On some devices its possible to overwrite the limites.
My g4 could go up to 60sec expotime, and my p9 can use iso 25k, also if both phones reported lesser values.
there is a freed_config.xml in Android/data/troop.com.freedcam/files wich can get edited

greets

**Reply by yzak4815** [reply]
Coool i'll try to modify tye xml thanks a Lot ;)

**Reply by yzak4815** [reply]
I mean the buttons in manual mode, the others are perfectly sized
